src/Hint/Naming.hs

```diff
@@ -49,9 +49,9 @@ namingHint :: DeclHint
 namingHint _ modu = naming $ Set.fromList $ concatMap getNames $ moduleDecls modu
 
 naming :: Set.Set String -> Decl_ -> [Idea]
-naming seen x = [suggestN "Use camelCase" x2 (replaceNames res x2) | not $ null res]
+naming seen x = [suggest "Use camelCase" x x2 [Replace Bind (toSS x) [] (prettyPrint x2)] | not $ null res]
```

Comment:
Before I think this suggesting replacing top-level names. Now I think it might suggest all recursive names? That seems like too much, since having internal names be whatever you want seems reasonable. Or am I misreading?

Reply:
Apologies, I'm not sure what you mean exactly - I'm guessing this changes because of `descendBi`?